src/main/scala/com/linagora/openpaas/gatling/Configuration.scala

```diff
@@ -33,6 +33,8 @@ object Configuration {
 
   val pkceCodeChallengeMethod = "S256"
 
+  val inboxSpaPath= Properties.envOrElse("INBOX_SPA_PATH", "inbox")
```

Comment:
```suggestion
  val InboxSpaPath = Properties.envOrElse("INBOX_SPA_PATH", "inbox")
```

since it is a constant